bin/land.js

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const childProcess = require('child_process');
+const https = require('https');
+
+const arg = process.argv.slice(2);
+const landedBranch = arg[0];
+
+const differCommits = () =>
+  parseInt(childProcess.execSync(`git rev-list --count master..HEAD`));
+
+const supportCommits = () => {
+  const git = childProcess.spawn('git', [
+    'rebase',
+    '-i',
+    '--autosquash',
+    `HEAD~10`,
+  ]);
+  process.stdin.setRawMode(true);
+  process.stdin.pipe(git.stdin);
+  git.stdout.pipe(process.stdout);
+  git.stdout.on('close', code => {
+    if (code !== 0) {
+      process.exit(code);
+    }
+  });
+};
+
+const supportRebase = () => {
+  const git = childProcess.spawn('git', [
+    'rebase',
+    '-i',
+    `HEAD~${differCommits()}`,
+  ]);
+  process.stdin.setRawMode(true);
+  process.stdin.pipe(git.stdin);
+  git.stdout.pipe(process.stdout);
+  git.stdout.on('close', code => {
+    if (code !== 0) {
+      process.exit(code);
+    }
+  });
+};
+
+const supportSquashAll = () => {
+  childProcess.execSync(`git reset --soft HEAD~${differCommits()}`);
+  const git = childProcess.spawn('git', ['commit']);
```

Comment:
This is almost the same in `supportCommits`, `supportRebase`, `supportSquashAll` except git arguments. It will be better to extract this to a separate function.